src/vorta/utils.py

```diff
@@ -318,3 +320,35 @@ def is_system_tray_available():
         is_available = tray.isSystemTrayAvailable()
 
     return is_available
+
+
+def validate_passwords(firstPass, secondPass):
+    passEqual = firstPass == secondPass
+    passLong = len(firstPass) > 8
+
+    if not passLong and not passEqual:
+        return trans_late('utils', "Passwords must be identical and greater than 8 characters long.")
+    if not passEqual:
+        return trans_late('utils', "Passwords must be identical.")
+    if not passLong:
+        return trans_late('utils', "Passwords must be greater than 8 characters long.")
+
+    return ""
+
+
+def password_transparency(encryption):
+    if encryption != 'none':
+        keyringClass = get_keyring().__class__.__name__
+        messages = {
+            'VortaDBKeyring':
+            trans_late('utils', 'plaintext on disk.\n'
```

Comment:
Suggested text for DB fallback could be like this: "Saving password in Vorta settings database. To store password more securely install a supported secret store."

Reply:
(This would be the description text in the `VortaDBKeyring` class)

Reply:
I prefer not to mention databases, just how its inseecure.

Reply:
Then "Saving password with Vorta settings. To store password more securely install a supported secret store."

Reply:
That seems fine to me